catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -42,6 +42,20 @@ Implementation details
         <argument ref="queryResponseTransformers"/>
     </bean>
 
+    <reference-list id="listOfMessageValidators" interface="ddf.catalog.validation.MessageValidator"
+                    availability="optional">
+        <reference-listener bind-method="bindPlugin" unbind-method="unbindPlugin">
+            <bean class="org.codice.ddf.platform.util.SortedServiceList"/>
+        </reference-listener>
+    </reference-list>
+
+    <bean id="messageValidatorManager" class="org.codice.ddf.catalog.ui.query.cql.MessageValidatorManager">
+        <cm:managed-properties
```

Comment:
It doesn't look like this object has any managed properties (there's no corresponding `metatype.xml`).